tasks/less.js

```diff
@@ -126,6 +126,14 @@ module.exports = function(grunt) {
       }
     }
 
+    if (_.isFunction(options.sourceMapURL)) {
```

Comment:
probably. just figured since lodash had already been imported, would use that utility. would you prefer `typeof` ?